src/Symfony/Component/Notifier/Bridge/Primotexto/PrimotextoOptions.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Primotexto;
+
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+
+/**
+ * @author Samaël tomas <samael.tomas@gmail.com>
+ */
+final class PrimotextoOptions implements MessageOptionsInterface
+{
+    private array $options;
+
+    public function __construct(array $options = [])
+    {
+        $this->options = $options;
+    }
+
+    public function getRecipientId(): ?string
+    {
+        return null;
+    }
+
+    /**
+     * @return $this
+     */
+    public function campaignName(string $campaignName): static
+    {
+        $this->options['campaignName'] = $campaignName;
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function category(string $category): static
+    {
+        $this->options['category'] = $category;
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function date(int $timestamp): static
```

Comment:
What does date mean?

Reply:
I added comment to explain and renaming the function to be more explicit